scss/reviewCard.module.scss

```diff
@@ -5,6 +5,12 @@
   position: absolute;
 }
 
-.accordionItem {
+.accordionItemShow {
```

Comment:
best practice is to name css using BEM: http://getbem.com/introduction/

Reply:
I'll look into it and update the names.